Update stale documentation in overview.md (#4968)
nextflow-io/nextflow: docs/overview.md

}
```

> The above example defines two processes. Their execution order is not determined by the fact that the `blastSearch` process comes before `extractTopHits` in the script (it could also be written the other way around). Instead, execution order is determined by their sequence in the _workflow_. In that block, `blastSearch` comes first, so it will execute first. `extractTopHits` executes after, and uses the output from `blastSearch`.

Reviewer: ```suggestion
The above example defines two processes. Their execution order is not determined by the fact that the `blastSearch` process comes before `extractTopHits` in the script (it could also be written the other way around). Instead, execution order is determined by their _dependencies_ -- `extractTopHits` depends on the output of `blastSearch`, so `blastSearch` will be executed first, and then `extractTopHits`.
```
